BuildFFmpeg.bat

```diff
@@ -1,292 +1,292 @@
-@setlocal
-@echo off
-
-if "%1" == "/?" goto Usage
-if "%~1" == "" goto Usage
-
-:: Initialize build configuration
-set BUILD.ARM=N
-set BUILD.x86=N
-set BUILD.x64=N
-set BUILD.win7=N
-set BUILD.win10=N
-set BUILD.win8.1=N
-set BUILD.phone8.1=N
-
-:: Export full current PATH from environment into MSYS2
-set MSYS2_PATH_TYPE=inherit
-
-:: Iterate through arguments and set the right configuration
-for %%a in (%*) do (
-    if /I "%%a"=="ARM" (
-        set BUILD.ARM=Y
-    ) else if /I "%%a"=="x86" (
-        set BUILD.x86=Y
-    ) else if /I "%%a"=="x64" (
-        set BUILD.x64=Y
-    ) else if /I "%%a"=="win7" (
-        set BUILD.win7=Y
-    ) else if /I "%%a"=="win10" (
-        set BUILD.win10=Y
-    ) else if /I "%%a"=="win8.1" (
-        set BUILD.win8.1=Y
-    ) else if /I "%%a"=="phone8.1" (
-        set BUILD.phone8.1=Y
-    ) else (
-        goto Usage
-    )
-)
-
-:: Set build all architecture if none are specified
-if %BUILD.ARM%==N (
-    if %BUILD.x86%==N (
-        if %BUILD.x64%==N (
-            set BUILD.ARM=Y
-            set BUILD.x86=Y
-            set BUILD.x64=Y
-        )
-    )
-)
-
-:: Set build all platform if none are specified
-if %BUILD.win10%==N (
-    if %BUILD.win8.1%==N (
-        if %BUILD.phone8.1%==N (
-            if %BUILD.win7%==N (
-                goto Usage
-            )
-        )
-    )
-)
-
-:: Verifying ffmpeg directory
-echo Verifying ffmpeg directory...
-pushd %~dp0\ffmpeg
-if not exist configure (
```

Comment:
This will not work if the script is not invoked from the local directory. %~dp0 references the folder from the absolute path.